src/clib/gaussj_g.cpp

```diff
@@ -0,0 +1,76 @@
+#include <numeric>
+#include <vector>
+
+#include "gaussj_g.hpp"
+
+namespace grackle::impl {
+
+int gaussj_g(int n, double* coef_matrix_fortran, double* vector) {
+
+    // TODO: to be removed
+    // Copy the matrix to a C-style layout (column-major order)
+    std::vector<double> coef_matrix(n*n);
+    for (int i = 0; i < n; ++i) {
+        for (int j = 0; j < n; ++j) {
+        coef_matrix[j*n + i] = coef_matrix_fortran[i*n + j];
+        }
+    }
+
+    // Loop over columns
+    for(unsigned int col=0; col<n; col++) {
```

Comment:
Some of the continuous integration is failing because it identifies a few places like this one, where `n` is unsigned and the index variable is signed. You should either:
- copy `n` into a temporary unsigned integer variable and use that new variable for your loop bounds, OR
- make the index-variable a signed integer in each case